tutorials/llama_index_knowledge_base_tutorial.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>what kind of knowledge base is this. why is it in a file already etc. I'm not sure we should hide this bit of code away - it might confuse the user. Similar to what @rogeryang was saying, placing some of the work you did in the notebook itself will make this tutorial more useful - e.g. optimize for copy/paste based development. </p><p> </p><p>As a consumer of this notebook, I don't know how to create this database index file. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I don't remember how long exactly, but creating the database index took a pretty long time and cost a couple dollars. Not sure it's something we want to run in the notebook every time.</p>